packages/async-ssr-manager/src/__tests__/index.test.ts

```diff
@@ -1,42 +1,30 @@
 // tslint:disable:no-implicit-dependencies
 
-import {
-  FeatureAppEnvironment,
-  FeatureServiceBinder,
-  FeatureServiceProviderDefinition
-} from '@feature-hub/core';
+import {FeatureAppEnvironment, FeatureServiceBinder} from '@feature-hub/core';
 import mockConsole from 'jest-mock-console';
 import {
   AsyncSsrManagerConfig,
   AsyncSsrManagerV0,
-  ServerRequest,
-  defineAsyncSsrManager
+  asyncSsrManagerDefinition
 } from '..';
 import {useFakeTimers} from './use-fake-timers';
 
-describe('defineAsyncSsrManager', () => {
+describe('asyncSsrManagerDefinition', () => {
   let mockEnv: FeatureAppEnvironment<AsyncSsrManagerConfig, {}>;
-  let asyncSsrManagerDefinition: FeatureServiceProviderDefinition;
-  let serverRequest: ServerRequest;
 
   beforeEach(() => {
     mockEnv = {
       config: {timeout: 5},
       featureServices: {},
       idSpecifier: undefined
     };
-
-    serverRequest = {
-      path: '/app',
-      cookies: {},
-      headers: {}
-    };
-
-    asyncSsrManagerDefinition = defineAsyncSsrManager(serverRequest);
   });
 
-  it('creates an Async SSR Manager definition', () => {
+  it('defines an id', () => {
     expect(asyncSsrManagerDefinition.id).toBe('s2:async-ssr-manager');
+  });
+
+  it('has no dependencies', () => {
```

Comment:
shouldn't this be also done for optionals?

Reply:
done